code/datums/keybinding/communication.dm

```diff
@@ -11,7 +11,8 @@
 	. = ..()
 	if(.)
 		return
-	winset(user, null, "command=[user.tgui_say_create_open_command(SAY_CHANNEL)]")
+	winset(user, null, "command=[user.tgui_say_create_open_command(SAY_CHANNEL)];")
+	winset(user, "tgui_say.browser", "focus=true")
```

Comment:
surely there's a way to join these statements but the documentation is too cryptic for me to grasp